config/initializers/ahoy.rb

```diff
@@ -6,7 +6,7 @@
 module Ahoy
   class Store < Ahoy::Stores::LogStore
     def exclude?
-      return if FeatureManagement.enable_load_testing_mode?
+      return true if FeatureManagement.enable_load_testing_mode?
```

Comment:
That is not correct. I explicitly made this change to allow events to be emitted during load testing for 2 reasons: 1) It allows us to confirm that the script is making the right requests in the right order, and 2) It better simulates the app load since emitting events is something we expect to happen in production.